README.md

```diff
@@ -27,6 +27,10 @@ This will help make clear the data requirements and how to structure this data t
 - Sam Abbott (seabbs)
 - Damon Bayer (damonbayer)
 
+# Package workflow
+The following depicts the suggested workflow for fitting the wastewater-informed forecasting model. See the ["Getting Started" vignette](vignettes/wwinference.Rmd) for a full example.
```

Comment:
I would add the link to the HTML document instead. Right now, it is not linking to the proper place on the website.

```suggestion
The following depicts the suggested workflow for fitting the wastewater-informed forecasting model. See the ["Getting Started" vignette](https://cdcgov.github.io/ww-inference-model/articles/wwinference.html) for a full example.
```